src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -563,5 +568,35 @@ export class OpenSheetMusicDisplay {
     public get Version(): string {
         return this.version;
     }
+
+    /**
+     * set player
+     * @param {(arr: Array<any>) => void} playCallback
+     */
+    public createPlayer(playCallback: (arr: Array<any>) => void): void {
+        this.player = new MusicSheetPlayer(80, this.sheet, this.cursor, playCallback);
```

Comment:
Why is this a special function and not initialized in the constructor?

Reply:
Thank you for your review. I've got all the suggestions and I'll modify my code until it's ready to push.